EnglishFilter/sections/specific.txt

```diff
@@ -180,6 +180,7 @@ oko.sh##a[href^="http://sundhopen.site/"]
 ||thecambabes.com/js/mypop2.js
 thecambabes.com##.list-videos > div.margin-fix > div.place
 thecambabes.com##.navigation > ul.primary > li > a:not([href*="thecambabes.com/"])
+||anyporn.com/js/v*_all.js$domain=anyporn.com,important
```

Comment:
That file is unblocked for adblock elements. Why block it again?

Overlay can be hidden fine with:
`anyporn.com##.inplb`